plugins/vppv2/model/punt/keys.go

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) 2018 Cisco and/or its affiliates.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package punt
+
+import (
+	"strconv"
+	"strings"
+)
+
+/* Punt */
+const (
+	// PrefixNAT44 is a key prefix used in NB DB to store configuration for NAT44.
+	Prefix = "vpp/config/v2/punt/"
+)
+
+/* Punt to Host */
+const (
+	// PrefixToHost is the key used in NB DB to store punt to host or unix domain socket configuration.
+	PrefixToHost = Prefix + "tohost/"
+
+	// toHostTemplate is the relative key prefix for punt to host.
+	toHostTemplate = PrefixToHost + "l3/{l3}/l4/{l4}/port/{port}"
+)
+
+/* IP redirect */
+const (
+	// PrefixIPRedirect is a key prefix used in NB DB to store punt IP redirect configuration.
+	PrefixIPRedirect = Prefix + "ipredirect/"
+
+	// ipRedirectTemplate is the relative key prefix for IP redirect.
+	ipRedirectTemplate = PrefixIPRedirect + "l3/{l3}/tx/{tx}"
+)
+
+const (
+	// InvalidKeyPart is used in key for parts which are invalid
+	InvalidKeyPart = "<invalid>"
+)
+
+/* Punt to host */
+
+// ToHostKey returns key representing punt to host/socket configuration.
+func ToHostKey(l3Proto L3Protocol, l4Proto ToHost_L4Protocol, port uint32) string {
+	strL3, strL4, strPort := strconv.Itoa(int(l3Proto)), strconv.Itoa(int(l4Proto)), strconv.Itoa(int(port))
```

Comment:
The numbers are gonna be confusing here. You can use generated function from this package to convert them to strings:

https://github.com/ligato/vpp-agent/blob/66c951f227fdf80d395037fe75dedb9920c18847/plugins/vppv2/model/punt/punt.pb.go#L31-L42